cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/WikipediaPipelineApp.java

```diff
@@ -29,15 +29,16 @@
  * App to demonstrate a data pipeline that processes Wikipedia data using a CDAP Workflow.
  */
 public class WikipediaPipelineApp extends AbstractApplication<WikipediaPipelineApp.WikipediaAppConfig> {
-  static final String PAGE_TITLES_STREAM = "pageTitleStream";
-  static final String RAW_WIKIPEDIA_STREAM = "wikiStream";
-  static final String PAGE_TITLES_DATASET = "pages";
-  static final String RAW_WIKIPEDIA_DATASET = "wikidata";
-  static final String NORMALIZED_WIKIPEDIA_DATASET = "normalized";
-  static final String SPARK_CLUSTERING_OUTPUT_DATASET = "clustering";
-  static final String MAPREDUCE_TOPN_OUTPUT = "topn";
-  static final String LIKES_TO_DATASET_MR_NAME = "LikesToDataset";
-  static final String WIKIPEDIA_TO_DATASET_MR_NAME = "WikiDataToDataset";
+  public static final String PAGE_TITLES_STREAM = "pageTitleStream";
+  public static final String RAW_WIKIPEDIA_STREAM = "wikiStream";
+  public static final String PAGE_TITLES_DATASET = "pages";
+  public static final String RAW_WIKIPEDIA_DATASET = "wikidata";
+  public static final String NORMALIZED_WIKIPEDIA_DATASET = "normalized";
+  public static final String SPARK_CLUSTERING_OUTPUT_DATASET = "clustering";
+  public static final String MAPREDUCE_TOPN_OUTPUT = "topn";
+  public static final String LIKES_TO_DATASET_MR_NAME = "LikesToDataset";
+  public static final String WIKIPEDIA_TO_DATASET_MR_NAME = "WikiDataToDataset";
+  static final String NAMESPACE_ARG = "namespace";
```

Comment:
Which code part are using the the ones which are made public ? I don't see any out of package usage of these in cdap-repo. Is it integration test-repo ? Can you point out ?